packages/peregrine/lib/talons/ProductFullDetail/useProductFullDetail.js

```diff
@@ -282,13 +286,31 @@ export const useProductFullDetail = props => {
         sku: product.sku
     };
 
+    const derivedErrorMessage = useMemo(() => {
+        const errorTarget = addSimpleError || addConfigurableError;
+        if (!errorTarget) return null;
+        if (errorTarget.graphQLErrors && errorTarget.graphQLErrors.length) {
+            // Apollo prepends "GraphQL Error:" onto the message,
+            // which we don't want to show to an end user.
+            // Build up the error message manually without the prepended text.
+            return errorTarget.graphQLErrors
+                .map(({ message }) => message)
+                .join(', ');
+        }
+        return errorTarget.message;
+    }, [addConfigurableError, addSimpleError]);
+
     return {
         breadcrumbCategoryId,
+        derivedErrorMessage,
```

Comment:
I can tolerate `derivedErrorMessage` as a variable name...but not really as a property name on a talon's return object. Maybe just `errorMessage` here?

Reply:
Oh definitely. Let me fix this.